HTML:CSS learning project/position.js

```diff
@@ -1,22 +1,24 @@
-$( document ).ready(function() {
-    $.fn.followTo = function (pos) {
-        var $this = this,
-            $window = $(window);
-    
-        $window.scroll(function (e) {
-            if ($window.scrollTop() > pos) {
-                $this.css({
-                    position: 'absolute',
-                    top: pos
-                });
-            } else {
-                $this.css({
-                    position: 'fixed',
-                    top: 0
-                });
-            }
-        });
-    };
-    
-    $('#block-menu').followTo(683);
+$(document).ready(function () {
+  $.fn.followTo = function (pos) {
+    var $this = this,
+      $window = $(window);
+
+    $window.scroll(function (e) {
+      $window.scrollTop() > pos
+        ? $this.css({
+            position: "absolute",
+            top: pos,
+          })
+        : $this.css({
+            position: "fixed",
+            top: 0,
+          });
+    }).on("resize", function(){ // If the user resizes the window
+        pos = $(this).height() - 60;
+    });
+        
+  };
+  var $win = $(window);
+  var winH = $win.height();
```

Comment:
bad naming of the variable. It's not clear what it is, better name it windowHeight instead. Also, use const/let and not var for declaring variables.